src/Uno.UI/UI/Xaml/Media/Animation/Animators/GPUFloatValueAnimator.iOSmacOS.cs

```diff
@@ -5,7 +5,13 @@
 using System.Text;
 using Windows.Foundation;
 using Uno.UI.DataBinding;
+#if __IOS__
 using UIKit;
+using NativeView = UIKit.UIView;
```

Comment:
Personally, when I use type aliases, I prefix the alias by `_` as it's a _private_ type. I found that much easier to understand when reading the code, especially when multi targeting code like here, it makes clear that we are dealing with a native object.

For instance here I would have defined:
```csharp
using _View = UIKit.UIView;
```

But FYI this is not an enforced rule, so it's your choice.

Reply:
Fixed :-)